test/system/src/test/java/io/pravega/test/system/KeyValueStoreTest.java

```diff
@@ -0,0 +1,744 @@
+import java.util.concurrent.*;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang.RandomStringUtils.randomAlphanumeric;
+import static org.junit.Assert.*;
+
+@Slf4j
+@RunWith(SystemTestRunner.class)
+public class KeyValueStoreTest extends AbstractSystemTest {
+
+    private static final String SCOPE_NAME = "SampleScope3" + randomAlphanumeric(5);
+    private static final String KVT_NAME = "TestKVT1";
+    private static final KeyValueTableInfo KVT = new KeyValueTableInfo(SCOPE_NAME,KVT_NAME);
+    private static final KeyValueTableConfiguration config = KeyValueTableConfiguration.builder().partitionCount(2).build();
+    private static final Serializer<Integer> KEY_SERIALIZER = new IntegerSerializer();
+    private static final Serializer<String> VALUE_SERIALIZER = new UTF8StringSerializer();
+    private URI controllerURI = null;
+    private Controller controller = null;
+    private KeyValueTableFactory keyValueTableFactory;
+    private KeyValueTable<Integer, String> keyValueTable;
+    private ConnectionFactory connectionFactory;
+    private Service controllerInstance;
+
+    @Before
+    public void setup() throws Exception {
+        controllerInstance = Utils.createPravegaControllerService(null);
+        List<URI> ctlURIs = controllerInstance.getServiceDetails();
+        final List<String> uris = ctlURIs.stream().filter(ISGRPC).map(URI::getAuthority).collect(Collectors.toList());
+        controllerURI = URI.create("tcp://" + String.join(",", uris));
+        final ClientConfig clientConfig = Utils.buildClientConfig(controllerURI);
+        final ScheduledExecutorService controllerExecutor= Executors.newScheduledThreadPool(5);
+        this.controller = new ControllerImpl(ControllerImplConfig.builder().clientConfig(clientConfig).build(),controllerExecutor);
+        this.connectionFactory = new ConnectionFactoryImpl(clientConfig);
+        boolean createdScope = this.controller.createScope(SCOPE_NAME).join();
+        assertTrue("The requested Scope must get created", createdScope);
+        log.info("Scope {} created successfully", SCOPE_NAME);
+        this.keyValueTableFactory = new KeyValueTableFactoryImpl(SCOPE_NAME,this.controller,this.connectionFactory);
+        boolean created = this.controller.createKeyValueTable(KVT.getScope(), KVT.getKeyValueTableName(), config).join();
+        Assert.assertTrue("The requested KVT must get created",created);
+        log.info("KVT {} created successfully", KVT_NAME);
+        this.keyValueTable = this.keyValueTableFactory.forKeyValueTable(KVT.getKeyValueTableName(), KEY_SERIALIZER, VALUE_SERIALIZER,
+                KeyValueTableClientConfiguration.builder().build());
+
+    }
+
+    @After
+    public void tearDown() {
+        this.connectionFactory.close();
+        this.controller.close();
+        this.keyValueTableFactory.close();
+        this.keyValueTable.close();
+    }
+
+    // Test case-42: Insert Same Key into multiple KeyFamilies
```

Comment:
use Javadoc

Reply:
Fix this here and for all methods below. Same request applies for all other comments I made.

Reply:
Sure, will change as suggested.